modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -185,6 +184,17 @@ class FeatureFloodCount : public GeneralPostprocessor,
      */
     bool boundingBoxesIntersect(const FeatureData & rhs) const;
 
+    /**
+     * The routine called to see if two features are mergable:
```

Comment:
I don't know how "mergeable" is actually spelled, but the documentation spelling should at least agree with the code spelling.